src/components/Auth/Logout.react.js

```diff
@@ -25,13 +28,14 @@ class Logout extends Component {
   }
 
   componentDidMount() {
+    const { actions, history } = this.props;
+    actions.logout();
     deleteCookie('loggedIn', { domain: cookieDomain, path: '/' });
     deleteCookie('serverUrl', { domain: cookieDomain, path: '/' });
     deleteCookie('emailId', { domain: cookieDomain, path: '/' });
     deleteCookie('showAdmin', { domain: cookieDomain, path: '/' });
```

Comment:
Not required, as we do not use this cookie now.

Reply:
@akshatnitd Oh right, missed that. I have removed it now. :+1: